### PR TITLE
Issue #1942 remove linting error ignores

### PR DESCRIPTION
**Pull request by Elnaril**
### What was wrong?
PR #1940 added the flake8 error code E252 to be ignored, which was a temporary solution to wait until all these errors were fixed in the codebase.

### How was it fixed?
 - E252 has been removed from the ignore section of flake8 in tox.ini
 - All errors raised by tox were fixed by adding a space around operators where missing

### To-Do

[//]: # (Stay ahead of things, add list items here!)
- [ ] Clean up commit history

[//]: # (For important changes that should go into the release notes please add a newsfragment file as explained here: https://github.com/ethereum/py-evm/blob/master/newsfragments/README.md)

[//]: # (See: https://py-evm.readthedocs.io/en/latest/contributing.html#pull-requests)
- [ ] Add entry to the [release notes](https://github.com/ethereum/py-evm/blob/master/newsfragments/README.md)

#### Cute Animal Picture

put a cute animal picture link inside the parentheses

### CODE REVIEW

**Comment by Elnaril** [comment]
Hello @pipermerriam , @carver , @cburgdorf , @gsalgado  , I'm not sure who I should ask for a review so I picked up the top 4 contributors ! :sweat_smile: :smile:  Would it be possible to have this PR reviewed ? :pray:

File: tox.ini [hunk 1]
```diff
@@ -10,9 +10,6 @@ envlist=
 max-line-length= 100
 exclude=
 ignore =
-    # E252 missing whitespace around parameter equals
-    # Already used too much in the codebase at the point of introduction.
-    E252,
     # W503 line break before binary operator
     # It's either this or W504 (line break _after_ binary operator), pick your poison.
     # W503 gets enabled when E252 gets ignored, see:
```
Comment (carver) [inline tox.ini:15]: W503 was only here because of E252, so we can actually delete the W503 ignore now, also.

**Review by carver — APPROVED** [review]
Yup, looks great, thanks for the contribution!

Would you mind stripping out the W503 also? While you're working on that, it seems that the two commits can be squashed too. But if you prefer, I'm happy to squash-merge it for you after the W503 change.

**Comment by Elnaril** [comment]
@carver  thanks for the review ! :-)

> W503 was only here because of E252, so we can actually delete the W503 ignore now, also.

Actually I tried it but I'm not sure that's possible because when I comment out this flag:
image

flake8 gets grumpy:
image

and if I add the line break after the operator, from:
image

to:
image

I get the W504:
image

**Comment by carver** [comment]
Interesting! I wonder if the commented out lines look to flake8 like an empty list that overrides the default list, which causes the same problem as adding E252. Anyway, no need to hunt that down right now. I'll just squash-merge unless you prefer to squash it yourself.

**Comment by Elnaril** [comment]
Yep, I agree the behaviour seems weird ...

> I'll just squash-merge unless you prefer to squash it yourself.

That's all good to me if you do it, thanks ! :-)

**Comment by Elnaril** [comment]
Actually, you're may be right about the comment (or empty include) confusing flake8:

with that:

image

the linting is ok:
image

but with that:
image

the errors are back ...

**Comment by Elnaril** [comment]
Shall I remove the `include=` and commit ?

**Comment by carver** [comment]
Hm, seems like that confusing error will just come back the next time someone wants to include an exception.

**Comment by carver** [comment]
Maybe change the comment like:
```diff
- W503 gets enabled when E252 gets ignored, see:
+ W503 gets enabled when any "ignore =" config is added (even the empty one), see:
```
and then I'll squash-merge.

**Comment by Elnaril** [comment]
Change is done

**Pull request merged**